service/src/main/java/org/cbioportal/genome_nexus/service/MutationAssessorService.java

```diff
@@ -0,0 +1,23 @@
+package org.cbioportal.genome_nexus.service;
+
+import org.cbioportal.genome_nexus.model.MutationAssessor;
+import org.cbioportal.genome_nexus.model.VariantAnnotation;
+import org.cbioportal.genome_nexus.service.exception.MutationAssessorNotFoundException;
+import org.cbioportal.genome_nexus.service.exception.MutationAssessorWebServiceException;
+import org.cbioportal.genome_nexus.service.exception.VariantAnnotationNotFoundException;
+import org.cbioportal.genome_nexus.service.exception.VariantAnnotationWebServiceException;
+
+import java.util.List;
+
+public interface MutationAssessorService
```

Comment:
Added an interface for `MutationAssessorService`, previously we only had an implementation class